src/lib/prepareOas.ts

```diff
@@ -211,6 +211,34 @@ export default async function prepareOas(
     debug('spec bundled');
   }
 
+  // Checks to see if the selected file is a postman collection and throws a warning in the console depending on the command.
+  if (definitionVersion.specification === 'postman') {
+    switch (command) {
+      case 'openapi':
+        warn(
+          'You are attempting to upload a Postman collection. This feature is currently experimental. For more information, visit our docs here: https://docs.readme.com/main/docs/openapi#the-api-reference',
```

Comment:
small refactoring suggestion: mind creating a separate variable above (called `postmanWarning` or something like that) and re-using that in all of these `warn` statements?